core-web/.husky/pre-commit

```diff
@@ -3,6 +3,5 @@
 
 cd core-web
 npx pretty-quick --staged
```

Comment:
Why remove the write?

Reply:
Because both used `prettier`, and that is a theme there, because, not existing a `.prettier` file in the root of core-web, I think... is the IDE fix all the `.prettier` (use the .prettier file from the core root folder). 
`npx pretty-quick --staged` fix all the touched files and `nx format:writer `format all the repo (I think... is better use format).
I think we need an issue to fix the .prettier problem, move to the nx root the prettier file and standardized use `format:writte` in pre-commit (exist a f`ormat:check`, and usually is used in the CI/CD)

Reply:
Added the prettier ( I copied the .prettier of the core/.prettier)

Reply:
https://stackoverflow.com/a/57987769

Reply:
Interesting, then removed `format:write` and add` npx pretty-quick --staged`